emcc.py

```diff
@@ -3963,6 +3959,7 @@ def modularize():
   %(script_url_node)s
   return (%(src)s);
 })();
+globalThis.AudioWorkletModule = %(EXPORT_NAME)s;
```

Comment:
Should this line not still be only emitted if `AUDIO_WORKLET` is set?   (Also what about the comment, is it no longer relevant/true?)

Reply:
I don't have any more ideas about it.

But I did test through the MODULARIZE and non-MODULARIZE modes

Reply:
hi, which `if/else` statement? can not quick understand.

Reply:
The previous was doing: ` 'globalThis.AudioWorkletModule = Module;' if settings.AUDIO_WORKLET and settings.MODULARIZE else ''`

With the old code this line was not emitted at all unless `settings.AUDIO_WORKLET and settings.MODULARIZE`.. presumably we would like to continue to skip this line unless these are both set?

Reply:
@sbc100 thanks, already fix that, just define globalThis.AudioWorkletModule when settings.AUDIO_WORKLET and settings.MODULARIZ.